render_files.py

```diff
@@ -178,6 +180,8 @@ def main():
     aws_access_key_id = os.environ.get("aws_access_key_id", "accessKey1")
     aws_secret_access_key = os.environ.get("aws_secret_access_key", "verySecretKey1")
     # openshift_release_image = os.environ.get("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE")
+    skip_cert_verification = os.environ.get('SKIP_CERT_VERIFICATION', False)
```

Comment:
I think that the default should be True.
This way you can push this, even before we are changing the assisted-service

Reply:
The default must be secure, which is not to skip cert verification. Better fail when you forget to turn it, then to stay insecure without noticing it. It is safe to push it this way because it does nothing with HTTP.